hedera-node/hapi-fees/build.gradle.kts

```diff
@@ -18,15 +18,6 @@ plugins { id("com.hedera.hashgraph.conventions") }
 
 description = "Hedera Services API Fees"
 
-configurations.all {
-  exclude("javax.annotation", "javax.annotation-api")
```

Comment:
We still do not want to have the Null annotations from java.annotations on the compile path to not import the wrong one. Maybe com.google.code.findbugs:jsr305 should be excluded now.